core/src/IRModel.ts

```diff
@@ -113,7 +116,7 @@ export class IRModel {
 
     const entities = await this._loader.getEntities();
     let relationships: IRRelationship[] = [];
-    if (typeof this._loader.getRelationships === "function")
+    // if (typeof this._loader.getRelationships === "function") ???
```

Comment:
should this commented if just be removed entirely? I'm not sure what all the relationships being set includes, but if we think that no wrong stuff is getting in there as a result of _loader.getRelationships() (since otherwise it would be empty anyway) then it should be fine to remove this comment

Reply:
```
public async getRelationships(): Promise<IRRelationship[]> { ... }
```

This is in the base class. The only thing a supertype can do is narrow the return type, right? So we can always expect `getRelationships` to return a function.

Reply:
That sounds right yeah, so i think we can remove that comment just to avoid future confusion